src/user/event_service.rst

```diff
@@ -0,0 +1,141 @@
+
+- When pilot starts EventService, here is how does a job process multiple events::
+
+    (pilot)
+    get_job
+    setup_job
+    while more_events:
+      get_event
+      process_event
+      update_event_status_to_panda
+    terminate_job
+    upload_logs
+
+EventService introduces a fine-grained scheduling level with bookkeepings. EventService is managed as
+part of a job. It has less scheduling overheads.
+
+- EventService is managed as part of a job. It's similart to Pilot 'PULL' job, in which the scheduling is
+  between PanDA and pilot. It skips Harvester, ARC CE and Cluster system like Slurm.
+
+- Comparing to job, EventService has one setup to setup the job, then run multiple events, and at the end
+  only transfer one log file for the job. It reduces the setup time, log transfering time and so on.
+
+
+EventService for Rubin
+----------------------
+
+EventService for ATLAS is mainly used for fine-grained simulation processing, in which it normally takes
+6~10 minutes to simulate an event. For Rubin, some jobs are too short which introduce huge overheads and
+loads in scheduling and logs transfering. EventService is one idea to reduce the oveheads and loads on
+different involved systems, without cutting the bookkeeping requirements in Rubin.
+
+How to map Rubin payloads to EventService:
+
+- During workflow submission time, define a mapping file to manage the mapping between event number and
+  the Rubin payloa. For example::
+
+    {
+    'pipeinit': {'0': 'pipeinit_0'},
+    'isr': {
+            '0': 'isr_payload_0',
+            '1': 'isr_payload_1',
+            ...
+            '1000': 'isr_payload_999'},
+     'final': {'0': 'final_payload_0'}
+     }
+
+- Here for example, EventService is enabled for 'isr' step, with 100 events per job. PanDA will manages
+  the event number 0~999 and split it to 10 jobs, job0: 0~99, job1: 100~199, ..., job9: 900~999.
+
+- Pilot starts a job, for example job1. Pilot will get an event from PanDA and will use the mapping file
+  to map the event number to a Rubin payload.
+
+How to run EventService
+~~~~~~~~~~~~~~~~~~~~~~~
+
+To enable EventService, users only need to define these values below in bps submission yaml files, which
+can be found in this file ${CTRL_BPS_PANDA_DIR}/config/bps_panda_eventservice.yaml.::
+
+  # enable EventService to have multiple payloads per PanDA job
+
+  # list of bps job labels to enable EventService
+  enableEventService: isr,step1
+
+  # When EventService is enabled, if requestWalltime and maxWmsJobWalltime are defined,
+  # max_payloads_per_panda_job = int(maxWmsJobWalltime / requestWalltime)
+  # Otherwise max_payloads_per_panda_job = int(maxPayloadsPerPandaJob)
+  maxWmsJobWalltime: 21600
+  maxPayloadsPerPandaJob: 10
+
+
+EventService monitor
+--------------------
+
+Event status can be shown in PanDA monitor, in the job status monitor page. Here is
+one example.
+
+.. image:: ../_images/event_service_monitor.png
+   :width: 6.5in
+   :height: 6.66667in
```

Comment:
Maybe show one with failure and retries on a single payload?

Also, maybe point out that bps report doesn't change (it doesn't include information at the PanDA job level, but the Rubin payload level).  Could include the corresponding output of `bps report --id <id>` for one of the above examples.

Reply:
I added (1) task level monitor, (2) job level monitor with event status which is already there, (3) bps report result.